frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,331 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Helmet } from 'react-helmet';
+import { match } from 'react-router-dom';
+import { ClusterServiceVersionLogo, iconFor } from './index';
+import {
+  Alert,
+  Bullseye,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  FlexModifiers,
+  Spinner,
+} from '@patternfly/react-core';
+import { Firehose, FirehoseResourcesResult } from '@console/internal/components/utils';
+import { k8sUpdate, referenceForModel } from '@console/internal/module/k8s';
+import {
+  ClusterServiceVersionModel,
+  InstallPlanModel,
+  PackageManifestModel,
+  SubscriptionModel,
+} from '../models';
+import {
+  CRDDescription,
+  ClusterServiceVersionKind,
+  SubscriptionKind,
+  InstallPlanKind,
+  PackageManifestKind,
+} from '../types';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@console/shared/src/components/status/icons';
+import { InstallPlanPreview } from './install-plan';
+
+const csvResources = (namespace: string) => [
+  {
+    kind: referenceForModel(ClusterServiceVersionModel),
+    namespaced: true,
+    fieldSelector: `metadata.name=${new URLSearchParams(window.location.search).get('pkg')}`,
+    isList: true,
+    namespace,
+    prop: 'clusterServiceVersions',
+  },
+  {
+    kind: referenceForModel(SubscriptionModel),
+    namespaced: false,
+    fieldSelector: `metadata.name=${new URLSearchParams(window.location.search).get('pkgname')}`,
+    isList: true,
+    optional: true,
+    prop: 'subscription',
+  },
+  {
+    kind: referenceForModel(InstallPlanModel),
+    prop: 'installPlan',
+    namespaced: true,
+    // fieldSelector: `spec.approval="Manual"`,
```

Comment:
I cannot make this fieldSelector work, although it is not a big deal if I do not filter the results here